discord/colour.py

```diff
@@ -329,7 +329,6 @@ def nitro_pink(cls: Type[CT]) -> CT:
         .. versionadded:: 2.0
         """
         return cls(0xF47FFF)
-
```

Comment:
This space (between `return cls(0xF47FFF)` and `@classmethod`) should still be here.